depthai_nodes/ml/parsers/scrfd.py

```diff
@@ -90,7 +93,7 @@ def build(
         SCRFDParser
             Returns the parser object with the head configuration set.
         """
-
+        super().build(head_config)
```

Comment:
Have you test SCRFD parser? When calling this `build` method from `DetectionParser` it should throw error that `Only one output layer supported for Detection, got {output_layer} layers.`, right?

Reply:
Yes I did test it but I used it directly and not via ParserGenerator (which uses build) as I thought nn_archives werent updated correctly. The actual parser works, now the build method should also work.